README.md

```diff
@@ -213,6 +213,27 @@ The default value is set to `lowerCase` and it will return a lower case name.
 By setting the value to `upperCase`, the words, will be returned with all the letters in upper case format.
 The `capital` option will capitalize each word of the unique name generated
 
+#### random
```

Comment:
This can be quite a useful feature. However this is already achievable since the user can provide the dictionaries to this library, therefore they can decide to just add random method to shuffle the dictionaries used for generating the final string.